examples/pylab_examples/custom_cmap.py

```diff
@@ -55,7 +67,30 @@
 never used.
 
 """
+# Make some illustrative fake data:
+
+x = np.arange(0, np.pi, 0.1)
+y = np.arange(0, 2*np.pi, 0.1)
+X, Y = np.meshgrid(x, y)
+Z = np.cos(X) * np.sin(Y) * 10
+
+
+# --- Colormaps from a list ---
 
+colors = [(1, 0, 0), (0, 1, 0), (0, 0, 1)]  # R -> G -> B
+n_bins = [2, 5, 10, 100]  # Discretizes the interpolation into bins
+cmap_name = 'my_list'
+f, axs = plt.subplots(1, 4, figsize=(10, 5))
```

Comment:
It looks like the colorbar mapping is somewhat different when doing it object-oriented style. If I do `im = imshow(...` and then call `fig.colorbat(im, cax=ax[0, 0])`, etc, then it gives a plot like this:

![image](https://cloud.githubusercontent.com/assets/1839645/16425473/bb033afa-3d19-11e6-9070-0c4c658974c5.png)

I think the colorbars are super tiny in the corners. Know how I can fix this?